analysis/clients/gossip.md

```
@@ -0,0 +1,24 @@
+# Gossip (Rust, Desktop)
+
+## Quick Facts
+| Setting | Value |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)